main/src/ca/uwaterloo/flix/runtime/shell/Command.scala

```diff
@@ -35,6 +35,11 @@ object Command {
     */
   case object Reload extends Command
 
+  /**
+    * Displays documentation
```

Comment:
Prints documentation about the definition or signature with fqn `s`.

Reply:
Done, but it does more than just definitions and signatures (it also handles enums and aliases).